subprojects/core/src/main/java/org/gradle/initialization/StartParameterBuildOptions.java

```diff
@@ -408,4 +409,23 @@ public void applyTo(StartParameterInternal settings, Origin origin) {
             settings.setExportKeys(true);
         }
     }
+
+    public static class InstantExecutionOption extends BooleanBuildOption<StartParameterInternal> {
+
+        public static final String PROPERTY_NAME = "org.gradle.unsafe.instant-execution";
+        public static final String LONG_OPTION = "instant-execution";
```

Comment:
we should include 'unsafe' in the command-line option name, if the system property name also has 'unsafe' in it.

Reply:
I aligned that with the file watching plan for introducing the experimental feature, for uniformity's sake.